### PR TITLE
[Doc Improvement][Update manifest schema configurable tabs for meeting side panel]

### PR DESCRIPTION
**Pull request by v-npaladugu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/bae7c3ab45579215fa79c5afac55d7746ace04fb/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 92 | 26 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2c49287f-aeca-4a4e-bc31-e8b4a1f5c4db) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/schema/manifest-schema.md [hunk 1]
```diff
@@ -456,7 +456,7 @@ Used when your app experience has a team channel tab experience that requires ex
 |`scopes`|array of enums|2|✔️|Currently, configurable tabs support only the `team` and `groupChat` scopes. |
 |`canUpdateConfiguration`|Boolean|||A value indicating whether an instance of the tab's configuration can be updated by the user after creation. Default: **true**.|
 |`meetingSurfaces`|array of enums|2||The set of `meetingSurfaceItem` scopes where a [tab is supported](../../tabs/how-to/access-teams-context.md). Default: **[sidepanel, stage]**. |
-|`context` |array of enums|8||The set of `contextItem` scopes where a [tab is supported](../../tabs/how-to/access-teams-context.md). Default: **[personalTab, channelTab, privateChatTab, meetingChatTab, meetingDetailsTab, meetingStage, callingSidepanel]**.|
+|`context` |array of enums|8||The set of `contextItem` scopes where a [tab is supported](../../tabs/how-to/access-teams-context.md). Default: **[personalTab, channelTab, privateChatTab, meetingChatTab, meetingDetailsTab, meetingSidePanel, meetingStage, callingSidepanel]**.|
```
Comment (ht4963-ms) [inline msteams-platform/resources/schema/manifest-schema.md:459]: As I mentioned before, this list is not the default. This is a list of acceptable values.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a8db81d80e6fac5f10f4da58efe7521c9cac6667/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 92 | 26 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a51aeb41-4855-4b61-bb77-2102c00c2c48) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by ht4963-ms — APPROVED** [review]

**Review by ht4963-ms — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/040a5853700d61f5dd3830c5aa184329521b9f55/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 92 | 26 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9f3c3d98-446f-4752-a42c-301a8be2caa9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**